### PR TITLE
Create project structure

### PR DESCRIPTION
**Pull request by sabahirfan**
This PR adds the basic structure for the project.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+8.9.4
```
Comment (timja-kainos) [inline .nvmrc]: is this the latest 8.x nodejs release?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "sinon-chai": "^3.0.0",
+    "mocha": "^5.0.3",
+    "mocha-junit-reporter": "^1.17.0",
+    "mocha-multi": "^1.0.1",
+    "mochawesome": "^3.0.2",
+    "tslint": "^5.5.0",
+    "sass-lint": "^1.10.2",
+    "tslint-config-standard": "^7.0.0"
+  }
+}
```
Comment (timja-kainos) [inline package.json]: Add this please:
```
  "publishConfig": {
    "access": "public"
  },
```
https://docs.travis-ci.com/user/deployment/npm/#Troubleshooting-%E2%80%9Cnpm-ERR!-You-need-a-paid-account-to-perform-this-action.%E2%80%9D

File: .nvmrc (hunk 1 above)
Comment (sabahirfan) [reply .nvmrc]: This is the one we have for our citizen-frontend. should I be using the latest here?

File: package.json (hunk 2 above)
Comment (sabahirfan) [reply package.json]: done

File: .nvmrc (hunk 1 above)
Comment (sabahirfan) [reply .nvmrc]: done

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+*.css
+
+.idea/
+*.iml
+
+*swp
+*log
+.DS_Store
+
+node-rpm-packaging/
```
Comment (timja-kainos) [inline .gitignore]: unneeded

**Pull request merged**